packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -87,6 +87,8 @@ def simcore_docker_compose(
     """
     COMPOSE_FILENAMES = ["docker-compose.yml", "docker-compose.local.yml"]
 
+    os.environ["DOCKER_REGISTRY"] = "local"
+    os.environ["DOCKER_IMAGE_TAG"] = "production"
     # ensures .env at git_root_dir

```

Comment:
IMO this is not a good idea for at least these two reasons:
 - fixture is modifying ``os.environ`` and therefore coupling to other tests
 - fixture might be overriding the settings of the tester without his knowledge
I would rather propose to set these environ variables ONLY if they are not defined AND with ``monkeypatch`` so their scope is well defined i.e.

```python
@pytest.fixture
def ffun(..., monkeypathc): 

   if "DOCKER_REGISTRY" not in os.environ:
       monkeypatch.setenv("DOCKER_REGISTRY", "local")
 ...

```

Reply:
what is monkeypatch?
ok for checking if. even though I doubt this is in use anywhere else. Also you specify in the comments in [docker_compose.py](https://github.com/sanderegg/osparc-simcore/blob/b274343598d4500690417819925221c26ee0cd5c/packages/pytest-simcore/src/pytest_simcore/docker_compose.py#L86) that it should do the same as `make .stack-simcore-version.yml` which is currently not the case

Reply:
then, we need to update the comments. Take into account that now this is an independent library

Reply:
[monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html#monkeypatching-mocking-modules-and-environments)

Reply:
ok this is done without monkeypatch (scope mismatch)